hack/generate/samples/internal/go/memcached-with-webhooks/e2e_test_code.go

```diff
@@ -288,15 +288,15 @@ var _ = Describe("memcached", func() {
 
 			By("validating that pod(s) status.phase=Running")
 			getMemcachedPodStatus := func() error {
-				cmd = exec.Command("kubectl", "get",
-					"pods", "-l", "app=memcached", "-o", "jsonpath={.items[*].status}",
-					"-n", namespace,
+				cmd =  exec.Command("kubectl", "get", "-l",
+					"app=memcached", "memcached",
```

Comment:
@NikhilSharmaWe we do not have the label app=memcahed with the plugin 
See that we will have `app.kubernetes.io/name=memcached` instead